docs/en/esc/esc_protocols.md

```diff
@@ -0,0 +1,66 @@
+# ESC Protocols
```

Comment:
FYI all same content, but have reorded to have Dshot and DroneCAN first - somewhat cooincidental, since I wnted them first and alphabetic ordering makes that happen. We should add Cyphal here